docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -115,82 +119,133 @@ POST /transactions/
+   :type fulfilled: boolean
+
+   :query owners_after: Public keys able to validly spend an output of a transaction, assuming the user also has the corresponding private key.
+   :type owners_after: base58 encoded string
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      GET /transactions?fields=id,conditions&fulfilled=false&owners_after=1AAAbbb...ccc HTTP/1.1
+      Host: example.com
+
    **Example response**:
 
    .. sourcecode:: http
 
-      HTTP/1.1 201 Created
+      HTTP/1.1 200 OK
       Content-Type: application/json
 
-      {
-        "id": "2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e",
-        "version": 1,
+      [{
         "transaction": {
           "conditions": [
             {
-              "amount": 1,
+              "cid": 0,
               "condition": {
                 "uri": "cc:4:20:GG-pi3CeIlySZhQoJVBh9O23PzrOuhnYI7OHqIbHjkk:96",
                 "details": {
                   "signature": null,
-                  "type_id": 4,
                   "type": "fulfillment",
+                  "type_id": 4,
                   "bitmask": 32,
-                  "public_key": "2ePYHfV3yS3xTxF9EE3Xjo8zPwq2RmLPFAJGQqQKc3j6"
+                  "public_key": "1AAAbbb...ccc"
                 }
               },
+              "amount": 1,
               "owners_after": [
-                "2ePYHfV3yS3xTxF9EE3Xjo8zPwq2RmLPFAJGQqQKc3j6"
-              ],
-              "cid": 0
-            }
-          ],
-          "fulfillments": [
-            {
-              "input": null,
-              "fulfillment": "cf:4:GG-pi3CeIlySZhQoJVBh9O23PzrOuhnYI7OHqIbHjkn2VnQaEWvecO1x82Qr2Va_JjFywLKIOEV1Ob9Ofkeln2K89ny2mB-s7RLNvYAVzWNiQnp18_nQEUsvwACEXTYJ",
-              "fid": 0,
-              "owners_before": [
-                "2ePYHfV3yS3xTxF9EE3Xjo8zPwq2RmLPFAJGQqQKc3j6"
+                "1AAAbbb...ccc"
               ]
             }
           ],
-          "operation": "CREATE",
-          "timestamp": "1477578978",
-          "asset": {
-            "updatable": false,
-            "refillable": false,
-            "divisible": false,
-            "data": null,
-            "id": "aebeab22-e672-4d3b-a187-bde5fda6533d"
-          },
-          "metadata": null
-        }
-      }
+        "id": "2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e",
+      }]
 
-   :statuscode 201: A new transaction was created.
-   :statuscode 400: The transaction was invalid and not created.
+   :resheader Content-Type: ``application/json``
 
+   :statuscode 200: A list of transaction's containing unfulfilled conditions was found and returned.
+   :statuscode 400: The request wasn't understood by the server, e.g. the ``owners_after`` querystring was not included in the request.
 
-GET /transactions/{tx_id}/status
---------------------------------
+.. http:get:: /transactions?fields=id,asset,operation&operation={GENESIS|CREATE|TRANSFER}&asset_id={asset_id}
```

Comment:
(proposal) we could shorten this query to 
``GET /transactions/assets/{asset_id}?fields=id,asset,operation&operation={GENESIS|CREATE|TRANSFER}``
or even
``GET /assets/{asset_id}?fields=id,asset,operation&operation={GENESIS|CREATE|TRANSFER}``
or (under the assumption `operation=CREATE`)
``GET /assets/{asset_id}?fields=id,asset,operation``

or maybe I'm being silly ;-)

anyhow, I know shortening should only be additional - and not core, but it might be useful for the end user to distinguish between the various flavors of transactions...